scripts/agency_fix.py

```
@@ -0,0 +1,364 @@
+            )
+            resp = client.chat.completions.create(
+                model=NVIDIA_MODEL,
+                messages=messages,  # type: ignore[arg-type]
+                temperature=0.1,
+                max_tokens=8192,
+            )
+            content = resp.choices[0].message.content if resp.choices else None
+            if content:
+                return content
+            log.warning("NVIDIA NIM returned empty content — falling back to Anthropic")
+        except Exception as exc:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)